api/src/data_migration/transformation/transform_oracle_data_task.py

```diff
@@ -89,6 +98,33 @@ def run_task(self) -> None:
             self.process_link_funding_categories()
             self.process_link_funding_instruments()
 
+    def _handle_delete(
+        self,
+        source: S,
+        target: D | None,
```

Comment:
I have no idea what's going on with this single letter vars 😵 I even looked into TypeVars briefly, and unfortunately, I still don't understand. But I did see that single letter vars are a common practice when defining TypeVars, so I don't think there's anything to change here.